src/pe_source/data/common_tlds.dict

```diff
@@ -0,0 +1,369 @@
+com
+org
+ru
+net
+de
+br
+uk
+it
+pl
+jp
+fr
+au
+ir
+in
+info
+cz
+nl
+es
+ca
+ua
+cn
+kr
+eu
+co
+ch
+gr
+tw
+za
+se
+ro
+hu
+at
+be
+mx
+vn
+ar
+sk
+dk
+no
+io
+tr
+me
+us
+cl
+biz
+fi
+tv
+pt
+nz
+ie
+xyz
+id
+il
+by
+hk
+club
+sg
+lt
+рф
+my
+online
+kz
+pro
+cc
+si
+hr
+bg
+th
+rs
+su
+pk
+az
+top
+site
+ng
+edu
+lv
+pe
+ae
+ee
+tk
+ph
+cat
+pw
+sa
+uz
+gov
+is
+mobi
+win
+nu
+lu
+ml
+asia
+ma
+lk
+shop
+ge
+ve
+live
+blog
+ws
+space
+ba
+ga
+uy
+bd
+cf
+website
+life
+tn
+ai
+tech
+md
+news
+mk
+ke
+do
+fm
+ec
+to
+am
+store
+mn
+la
+py
+coop
+media
+travel
+al
+world
+work
+np
+one
+eg
+guru
+cr
+kg
+link
+stream
+tz
+ly
+gg
+app
+icu
+nyc
+qa
+agency
+gt
+host
+network
+bid
+cloud
+vip
+bo
+cy
+dz
+im
+bz
+aero
+gq
+ug
+global
+design
+digital
+eus
+pa
+af
+vc
+academy
+li
+ao
+center
+jobs
+tj
+studio
+sv
+ovh
+mm
+jo
+ag
+click
+video
+sd
+kw
+zone
+zw
+moe
+press
+cm
+ps
+ci
+city
+ooo
+re
+ninja
+lb
+wiki
+mt
+om
+education
+company
+mz
+gh
+mu
+kh
+trade
+solutions
+mo
+church
+men
+group
+tips
+sy
+cu
+ni
+ac
+plus
+red
+sh
+guide
+rw
+games
+bh
+sn
+st
+hn
+art
+iq
+expert
+events
+cash
+social
+party
+school
+bike
+tools
+ltd
+services
+moscow
+et
+bank
+fo
+so
+market
+cool
+sexy
+team
+marketing
+ms
+tt
+zm
+bzh
+mg
+works
+bio
+bet
+bw
+cx
+land
+as
+community
+pg
+gal
+love
+pub
+cd
+bt
+email
+tm
+na
+systems
+date
+ink
+cafe
+gdn
+blue
+capital
+coffee
+sc
+pics
+bn
+chat
+run
+fit
+science
+help
+bf
+house
+укр
+care
+nc
+exchange
+porn
+photography
+bm
+audio
+loan
+scot
+gratis
+swiss
+watch
+ad
+kim
+support
+training
+рус
+gl
+gs
+technology
+codes
+gallery
+reviews
+wtf
+movie
+mv
+wien
+webcam
+farm
+money
+style
+wang
+ht
+jm
+software
+bar
+beer
+law
+photo
+int
+photos
+directory
+africa
+energy
+pf
+gi
+tc
+fund
+mw
+international
+fj
+lol
+amsterdam
+report
+camp
+pr
+earth
+pink
+sm
+pm
+ky
+ventures
+restaurant
+bi
+foundation
+mr
+je
+mc
+deals
+direct
+mil
+show
+tube
+careers
+film
+fyi
+buzz
+cam
+tl
```

Comment:
Is this data something we might consider putting into the database?

Reply:
The common_tlds.dict is just the default top level domain list that comes with DNSTWIST, i dont think its necessary to put it in the database.